include/quicr/quicr_client.h

```diff
@@ -201,7 +202,10 @@ class QuicRClient
                                   bytes&& data);
 
 protected:
-  std::unique_ptr<QuicRClientSession> client_session;
+  std::unique_ptr<ClientSession> client_session;
 };
 
+using QuicRClient
+  [[deprecated("quicr::QuicRClient stutters, use quicr::Client")]] =
```

Comment:
Is there a way to move this up a line, or does the annotation need to be right here?

Reply:
You can thank clang-format and punch cards here. Attributes for using statements have to come before the =, so unfortunately it looks like this. Thanks punch cards!